### DISCUSSION TITLE
Dr Grandet Mode 是什么？

### DISCUSSION THREAD

**Discussion by LipChuanSui** [MaaAssistantArknights/MaaAssistantArknights#7265] [Q&A] [answered]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56187131/92b3a28b-fe17-47d7-81cc-25db6d075d0c)

**Comment by Private233** [comment, marked as answer]
葛朗台模式，如果当前磕理智药将超过上限，那么可能会亏当前恢复的半点理智（6分钟回复1理智，嗑药超上限会导致6分钟重置）
开了这个模式会先等6分钟走完恢复上这1理智后再嗑药

**Reply by LipChuanSui** [reply]
thank you
